new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ExecResult.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2018, 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.utils;
+
+import java.io.IOException;
+
+/**
+ * Class that holds the results of using java to exec a command (i.e. exit value, stdout and stderr)
+ */
+public class ExecResult {
+  private int exitValue;
+  private String stdout;
+  private String stderr;
+
+  /**
```

Comment:
don't think we need javadoc for the constructor, it's pretty clear what it does

Reply:
The class was copied from existing integration tests. I'll remove the comment.